src/utils/popup-keyboard-accessibility.tsx

```diff
@@ -89,9 +89,11 @@ export const withKeyboardA11y = (WrappedComponent): any =>
      * @memberof HOC
      */
     componentWillUnmount(): void {
-      if (this._previouslyActiveElement) {
-        this._previouslyActiveElement.focus();
-      }
+        setTimeout(() => {
+          if (this._previouslyActiveElement && document.contains(this._previouslyActiveElement)) {
+            this._previouslyActiveElement.focus();
+          }
+        }, 100);
```

Comment:
if use default timeout `0` doesn't it work?

Reply:
No. I've tried and it's not working with timeout 0